### PR TITLE
Add description field to pokemon and api.service

### PR DESCRIPTION
**Pull request by AlexanderLill**
See https://github.com/PokemonGoers/Catch-em-all/issues/23#issuecomment-247795969.

Afaik this should be all necessary changes, right?

### CODE REVIEW

File: ionic2/app/services/api.service.ts
Comment (WoH) [inline ionic2/app/services/api.service.ts:81]: This is not backed by the API according to their docs.

Edit: At least not yet, I can see the commits on a feature branch, but let's not merge this until they are ready aswell.
I'm not gonna argue how useful searching by description is :laughing: 
But once the PokeData API supports it, this should be good to go.

**Review by WoH — CHANGES_REQUESTED** [review]

File: ionic2/app/services/api.service.ts
Comment (AlexanderLill) [reply ionic2/app/services/api.service.ts:81]: @WoH The PokeData team has all those changes in a pull request atm :)

**Review by WoH — APPROVED** [review]

File: ionic2/app/services/api.service.ts
Comment (WoH) [reply ionic2/app/services/api.service.ts:81]: @AlexanderLill See edit in the comment above, I'm not going to hold this back, but I'd suggest to wait anyway :wink:

Comment (AlexanderLill) [reply ionic2/app/services/api.service.ts:81]: @WoH I totally see your point in wating until it is merged :) I also dont see a use-case for searching for description, but we could aswell include it in the search bar. I think they just implemented it because ... who knows ... :)

**Pull request merged**
